plugins/modules/aci_l3out_hsrp_group.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_hsrp_group
+short_description: Manage HSRP group (hsrpGroupP) of the HSRP interface profile (hsrpIfP)
+description:
+- Manage HSRP group of the HSRP interface profile on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+    required: true
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+    required: true
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+    required: true
+  interface_profile:
+    description:
+    - Name of an existing interface profile.
+    type: str
+    aliases: [ name, interface_profile_name, logical_interface ]
+    required: true
+  hsrp_interface_group:
+    description:
+    - Name of the HSRP interface group.
+    type: str
+    aliases: [ name, hsrp_group ]
+  group_id:
+    description:
+    - The group id of the HSRP interface group.
+    type: int
+  ip:
+    description:
+    - The virtual IP address of the HSRP interface group.
+    type: str
+  mac:
+    description:
+    - The MAC address of the HSRP interface group.
+    type: str
+  group_name:
+    description:
+    - The group name of the HSRP interface group.
+    type: str
+  description:
+    description:
+    - The description of the HSRP interface group.
+    type: str
+    aliases: [ descr ]
+  group_type:
+    description:
+    - The type of the HSRP interface group.
+    type: str
+    choices: [ ipv4, ipv6 ]
+  ip_obtain_mode:
+    description:
+    - The mode of method used to obtain the IP address.
+    type: str
+    choices: [ admin, auto, learn ]
+  group_policy:
+    description:
+    - The group policy of the HSRP interface group.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+notes:
+- The C(tenant), C(l3out), C(logical_node_profile), C(logical_interface_profile) and C(hsrp_interface_profile) must exist before using this module in \
+  your playbook. The M(cisco.aci.aci_tenant), M(cisco.aci.aci_l3out), M(cisco.aci.aci_l3out_logical_node_profile), \
+  M(cisco.aci.aci_l3out_logical_interface_profile) and M(cisco.aci.aci_l3out_hsrp_interface_profile) can be used for this.
```

Comment:
will the \ impact the documentation hosted on galaxy? should we use regular newlines?

Reply:
I checked the galaxy docs and '\' is not required. Even if the notes is spread over lines, it will appear in a single line without '\'